Neos.Flow/Classes/Persistence/Doctrine/Service.php

```diff
@@ -641,27 +646,25 @@ class $className extends AbstractMigration
     /**
      * @return string
      */
-    public function getDescription(): string
+    public function getDescription()
```

Comment:
Return type removed? Was this breaking for some cases?

Reply:
It should match the signature from Doctrine… And in `doctrine/migrations` 1.8 that is without.